collections/indexes/multi.go

```diff
@@ -121,6 +123,40 @@ func (m *Multi[ReferenceKey, PrimaryKey, Value]) MatchExact(ctx context.Context,
 	return m.Iterate(ctx, collections.NewPrefixedPairRange[ReferenceKey, PrimaryKey](refKey))
 }
 
+// RefKeys returns a list of all the MultiIterator's reference keys (may contain duplicates).
+// Enable the "unique" argument to get a unique list of reference keys (the reference key must be comparable)
+func (m *Multi[ReferenceKey, PrimaryKey, Value]) RefKeys(ctx context.Context, unique bool) ([]ReferenceKey, error) {
```

Comment:
this should be documented as expensive call since it can end up iterating over a loooot of data.

Reply:
added the comment here: https://github.com/oren-lava/cosmos-sdk/commit/a56e94317477e96b81982f69e82989ea91adb699